requirements/prod.txt

```diff
@@ -102,8 +102,8 @@ bleach==1.5.0 \
 celery==3.1.23 \
     --hash=sha256:eaf5dee3becbc35c7754a2d4482d53bdf72ea3f85dd258525259983262081474 \
     --hash=sha256:1a359c815837f9dbf193a7dbc6addafa34612c077ff70c66e3b16e14eebd2418 # pyup: ==3.1.23
-certifi==0.0.8 \
-    --hash=sha256:46ecf5f7526a08cc1f8bc8232adf0cffce046f46ceff95539daec42ebc4849ef # pyup: ==0.0.8
+certifi==2017.4.17 \
+    --hash=sha256:f4318671072f030a33c7ca6acaef720ddd50ff124d1388e50c1bda4cbd6d7010
```

Comment:
Same here, 

We must not update certifi, there's a bug with it unfortunately and it breaks certificate related stuff on production. So this should be backed out.

Reply:
What do you think of adding a pip install just for certifi in the Makefile? I can't get around the ```Multiple .dist-info directories``` error.

Reply:
As a workaround, yeah, let's do that for now but mark it with a big `TODO` and open an issue to investigate alternatives.